dask/dask.yaml

```diff
@@ -2,6 +2,9 @@ temporary-directory: null  # Directory for local disk like /tmp, /scratch, or /l
 
 dataframe:
   shuffle-compression: null  # compression for on disk-shuffling. Partd supports ZLib, BZ2, SNAPPY, BLOSC
+  parquet:
+    metadata-task-size-local: 512  # Number of files per local metadata-processing task
+    metadata-task-size-remote: 16  # Number of files per remote metadata-processing task
```

Comment:
Okay - Did a fair bit of local testing to confirm that tese are reasonable defaults. Note that I also tweaked the logic so that the parallel metadata processing will not kick in unless the number of files exceeds this number.